packages/monitor-v2/src/monitor-og/README.md

```diff
@@ -0,0 +1,35 @@
+# Optimistic Governor Monitoring
+
+Optimistic Governor monitor bots can report on real time events based on the provided configuration for all the networks
+where `OptimisticGovernor` contract is deployed.
+
+The main entry point to Optimistic Governor monitor bots is running:
+
+```
+node ./packages/monitor-v2/dist/monitor-og/index.js
+```
+
+All the configuration should be provided with following environment variables:
+
+- `OG_ADDRESS` is the address of the deployed Optimistic Governor that this bot will monitor.
```

Comment:
I added this prop and changed how the bot works to have the address of the OG in question passed in as a prop.